rector.php

```diff
@@ -22,7 +35,22 @@
         naming: true
     )
     ->withPhpSets()
-    ->withRules([DeclareStrictTypesRector::class])
+    ->withRules([
+        DeclareStrictTypesRector::class,
+        BinaryOpNullableToInstanceofRector::class,
+        WhileNullableToInstanceofRector::class,
+        IntvalToTypeCastRector::class,
+        StrvalToTypeCastRector::class,
+        BoolvalToTypeCastRector::class,
+        FloatvalToTypeCastRector::class,
+        IncreaseDeclareStrictTypesRector::class,
+        StaticClosureRector::class,
+        StaticArrowFunctionRector::class,
+        PostIncDecToPreIncDecRector::class,
+        UseIncrementAssignRector::class,
+        ArraySpreadInsteadOfArrayMergeRector::class,
+        FinalizeTestCaseClassRector::class,
```

Comment:
@TomasVotruba there is other option for this, eg: add new set for collection of opiniated sets, but usefull for our code base, eg `rector-preset`, wdyt?

Reply:
I created new alternative PR for it:

- https://github.com/rectorphp/rector-src/pull/5924